docs/Cask-Maintainer-Guide.md

```diff
@@ -0,0 +1,59 @@
+# Cask Maintainer Guide
+
+This guide is intended to help maintainers effectively maintain the cask repositories.
+It is meant to be used in conjunction with the more generic [Maintainer Guidelines](Maintainer-Guidelines.md).
+
+This guide applies to all four of the cask repositories:
+
+- [Homebrew/homebrew-cask](https://github.com/Homebrew/homebrew-cask): The main cask repository
+- [Homebrew/homebrew-cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers): Casks of drivers
+- [Homebrew/homebrew-cask-fonts](https://github.com/Homebrew/homebrew-cask-fonts): Casks of fonts
+- [Homebrew/homebrew-cask-versions](https://github.com/Homebrew/homebrew-cask-versions): Alternate versions of Casks
+
+## Common Situations
+
+Here is a list of the most common situations that arise in PRs and how to handle them:
+
+- The `version` and `sha256` both change (keeping the same format): Merge
+- Only the `sha256` changes: Merge unless the version needs to be updated as well.
+  It’s not uncommon for upstream vendors to update versions in-place.
+- `livecheck` is updated: Use your best judgement and try to make sure that the changes
+  follow the [`livecheck` guidelines](Brew-Livecheck.md).
+- Only the `version` changes or the `version` format changes: Use your best judgement and
+  merge if it seems correct (this is relatively rare).
+- Other changes (including adding new casks): Use the [Cask Cookbook](Cask-Cookbook.md) to determine what's correct.
+
+If in doubt, ask another cask maintainer on GitHub or Slack.
+
+Note that unlike in formulae, casks do not consider the `sha256` stanza as meaningful security measure as maintainers cannot realistically check them for authenticity. Casks download from upstream; if a malicious actor compromised a URL, they could just as well compromise a version and make it look like an update.
+
+## Merging
+
+### Approvals
+
+In general, PRs in the cask repositories should have at least one approval from a maintainer
+before being merged.
```

Comment:
Can we make it a must?

Reply:
Fine with me, this is probably a question for the cask folks. I've not been poking around long enough to know how often self-approvals are done.

Reply:
Unless we’ve changed that when the bot was disabled, it’s impossible to merge without an approval (doesn’t pass CI).

Reply:
We might be talking about different things here.

Yes, an approval is necessary to merge a PR. However, this approval can come come be from @BrewTestBot (see, for example, https://github.com/Homebrew/homebrew-cask/pull/105694#pullrequestreview-658925897).

What I'm trying to get across here is that if possible, PRs should have an approval from _another_ maintainer. However, a maintainer can also use the self-approve feature to _allow_ for a PR of theirs to be merged without another maintainer.

Are you suggesting this be removed? Either way, I've updated the text to try to clarify this.

Reply:
The vast majority of changes are simple version bumps whose validity can be confirmed by CI. At the very least “in general” should be removed, because in general there’s really no need to enforce another maintainer looking at it.

Only situations where I can see another maintainer reviewing might be advantageous:

* Adding a cask.
* Renaming a cask.
* Removing a cask.

Pinging @miccal in case I’m forgetting something.

Reply:
I think I've reworked this. Feel free to unresolve or continue the discussion below if not.